dotCMS/src/main/java/com/dotmarketing/util/TaskLocatorUtil.java

```diff
@@ -402,6 +402,8 @@ public static List<Class<?>> getBackportedUpgradeTaskClasses() {
 	public static List<Class<?>> getStartupRunOnceDataTaskClasses() {
 		final List<Class<?>> ret = ImmutableList.<Class<?>>builder()
 				.add(Task230320FixMissingContentletAsJSON.class)
+				//TODO: Uncomment next line once we have the changes in portlet to stop saving LanguageVariables in the old way
```

Comment:
@nollymar This todo also breaks Sonarqube convention and shows as 1 new issue.   By default in sonarqubue TODOs are expected to be just for the developers to remind of stuff that needs to be cleaned up before the PR is created,  catching these can help to identify where something was missed.  We treat these more as long term TODOs for the product,   the question is whether this is good practice to use it like this, similar to the issue of commented out code.   or if we feel that due to our own conventions and existing code that using a TODO comment like this is ok we should specifically remove this rule from sonarqube so it does not become just noise.

Reply:
Changes to TaskLocatorUtil were reverted. I'll create a ticket to document that the UpgradeTask needs to be added or renamed and added in case something newer comes up